week3/day1/README.md

```diff
@@ -1,3 +1,5 @@
 # Git Review!!!
 
 This is a quick git review!!! Or is it?
+
+![](http://i.giphy.com/l3V0wpHLf2qOEwegE.gif)
```

Comment:
... Don't worry about it, it's necessary to add this line of code. Please add +1!!!